blocks/hero-block/features/hero/default.jsx

```diff
@@ -34,20 +41,95 @@ function Hero({ customFields }) {
 		subHeadline,
 		variant = "dark",
 	} = customFields;
-
+	const { arcSite } = useFusionContext();
+	const { fallbackImage, resizerURL } = getProperties(arcSite);
+	const imageId = imageDesktopURL
+		? imageDesktopURL.split("/").pop().split(".").shift()
+		: desktopImgId;
+	const imageMobileId = imageMobileURL
+		? imageMobileURL.split("/").pop().split(".").shift()
+		: imgMobileId;
+	let desktopAuth = useContent(
+		imageId
+			? {
+					source: "signing-service",
+					query: { id: imageId },
+			  }
+			: {}
+	);
+	const imageAuthTokenObj = {};
+	if (desktopAuth?.hash) {
+		desktopAuth[RESIZER_TOKEN_VERSION] = desktopAuth.hash;
+	}
+	if (desktopAuth && !Object.keys(desktopAuth).length) {
+		desktopAuth = imageDesktopAuth;
+	}
+	let mobileAuth = useContent(
+		imageMobileId
+			? {
+					source: "signing-service",
+					query: { id: imageMobileId },
+			  }
+			: {}
+	);
+	const imageMobileAuthTokenObj = {};
+	if (mobileAuth?.hash) {
+		mobileAuth[RESIZER_TOKEN_VERSION] = mobileAuth.hash;
+	}
+	if (mobileAuth && !Object.keys(mobileAuth).length) {
+		mobileAuth = imageMobileAuth;
+	}
 	const classes = [
 		BLOCK_CLASS_NAME,
 		`${BLOCK_CLASS_NAME}--${layout}`,
 		variant === "dark" ? `${BLOCK_CLASS_NAME}--dark` : `${BLOCK_CLASS_NAME}--light`,
 	].join(" ");
 
+	const alt = headline || description || null;
+	const desktopImageParams =
+		imageId && imageDesktopURL
+			? {
+					src: imageANSToImageSrc({
+						_id: imageId,
+						url: imageDesktopURL,
+					}),
+					resizedOptions: {
+						auth: (desktopAuth && JSON.parse(desktopAuth).hash) || imageAuthTokenObj,
```

Comment:
`desktopAuth` is already an object, so `JSON.parse` isn't needed and is causing errors.

Reply:
@vgalatro It's coming as an object. 
<img width="1656" alt="Screenshot 2023-05-09 at 11 23 03 AM" src="https://github.com/WPMedia/arc-themes-blocks/assets/83021791/0e6f6bdb-3b86-4da8-98ac-a1412aa2b982">

Reply:
`desktopAuth` is a stringified object not a Object hence we need to parse and extract the hash value out of it that will be passed down to `resizedOptions` auth key.

Reply:
I see what's happening, there's mixing of data going on since the auth set in PageBuilder editor, and the response from signing service are being set to the same variable. The auth string set in PageBuilder will always be a JSON string, and the response from the signing service will always be an object. There's an `imageAuthToken` variable that gets initialized, but never actually set so that should be removed.

Reply:
Fixed it.